### PR TITLE
feat: add style and bug

### PR DESCRIPTION
**Pull request by Turdubaevajazgul**
Issue #:
[Trello](https://trello.com/b/uAKK4mwb/solid-zillow)

## PR type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [x] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### before:
image

#### after:
image

## Does this introduce a breaking change?

- [x] Yes (please describe):
- [ ] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [x] Responsive layout and functionality have been tested
  - [x] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

**Comment by Bezymeys** [comment]
где скриншот?

File: src/components/card/Card.module.scss [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+        font-weight: 500;
+        font-size: 16px;
+        line-height: 24px;
+        overflow: visible;
+        margin-top: 10px;
+    }
+
+}
+
+@media(max-width:1030px) {
```
Comment (Bezymeys) [inline src/components/card/Card.module.scss]: image
не верная адаптивка

File: src/components/card/Card.module.scss [hunk 2]
```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700;500&display=swap');
+
+.container {
```
Comment (Bezymeys) [inline src/components/card/Card.module.scss:6]: image
почему у контейнера ширина 100%?
они должны располагаться в середине

**Review by Bezymeys — CHANGES_REQUESTED** [review]

File: src/components/card/CardLinks.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { Link } from "react-router-dom";
+
+export default function CardLink({ id }) {
+  return (
+    <div>
```
Comment (kairat-asylbekov-engineer) [inline src/components/card/CardLinks.jsx]: пустые дивы удаляем, тут вместо дива напишите реакт фрагмент

File: src/components/dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -1,9 +1,11 @@
 import React from "react";
+import Card from "../card/Card";
 
 function Dashboard() {
   return (
     <>
       <div>Dashboard</div>
```
Comment (kairat-asylbekov-engineer) [inline src/components/dashboard/Dashboard.jsx]: текста не пишем в тег div, для текстов есть свои специальные теги: p, h1, span

**Review by AiteginJyldyzbekov — APPROVED** [review]
LGTM!

**Pull request merged**